client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/CustomEventLogListener.java

```diff
@@ -238,6 +238,30 @@ private void expectTimeOrdered(CustomEvent before, CustomEvent after) {
       public DynamicCustomEventWithStateLogListener(RemoteCache<K, ?> r) { super(r); }
    }
 
+   @ClientListener(converterFactoryName = "simple-converter-factory")
+   public static class SimpleListener<K> extends CustomEventLogListener<K, String> {
+      public SimpleListener(RemoteCache<K, ?> r) {
+         super(r);
+      }
+   }
+
+   @NamedFactory(name = "simple-converter-factory")
+   static class SimpleConverterFactory<K> implements CacheEventConverterFactory {
+      @Override
+      @SuppressWarnings("unchecked")
+      public CacheEventConverter<String, String, CustomEvent> getConverter(Object[] params) {
+         return new SimpleConverter();
+      }
+
+      static class SimpleConverter<K> implements CacheEventConverter<K, String, String>, Serializable, ExternalPojo {
+         @Override
+         public String convert(K key, String oldValue, Metadata oldMetadata, String newValue, Metadata newMetadata, EventType eventType) {
+            if (newValue != null) return newValue;
```

Comment:
I prefer ```return newValue==null? oldValue:newValue;```

Reply:
This JIRA has 3 PRs, so I since this is only a test, let's keep it "as is"  :)